# Signal the gripper to use the force configuration q
caros_common/Q force
---
# Acknowledgement of the service request
bool success
